pyproject.toml

```diff
@@ -1,13 +1,16 @@
 [tool.black]
 line-length = 99
```

Comment:
99 columns of code on the wall. Would have liked black's shorter default but that would be an even bigger change.

Reply:
I'm ok with removing this, just say the word

Reply:
comparing...

Reply:
I wouldn't mind the shorter version, too

Reply:
I consider this decided then 😄